.github/workflows/nightly.yml

```diff
@@ -0,0 +1,21 @@
+name: nightly
+on:
+  schedule:
+    - cron: '0 1 * * *'
+  # can be run manually on https://github.com/rhinstaller/anaconda-webui/actions
+  workflow_dispatch:
+jobs:
+  trigger:
+    permissions:
+      statuses: write
+    runs-on: ubuntu-22.04
```

Comment:
Is there a specific reason for this? Unless there is, please use "ubuntu-latest". It's 24.04 now, and it'll just move with the time. There's nothing special going on here, just a little bit of Python.

Reply:
Nope replaced.